src/base/server_type.cpp

```diff
@@ -277,6 +289,26 @@ int setup_tran_server_params_on_ha_mode ()
   return NO_ERROR;
 }
 
+bool is_localhost (const char *hostname)
```

Comment:
why not reuse `util_is_localhost` -> `are_hostnames_equal` ?

Reply:
Thanks, I replace this function with `util_is_localhost`.